### ISSUE TITLE
Giada 0.17.2 "Sphinx don't open in Mojave

### ISSUE THREAD

**Issue by nay-seven** [monocasual/giada#458]
**Environment**
- OS: [e.g. macOS 10.14.6]
- Giada version: [e.g. v0.17.2]

**Describe the bug**
Message on startup " Check with the developer that giada works with this version of macOS. You may have to reinstall the application. "

**Comment by gvnnz** [comment]
Thanks for reporting. This is a known issue, started with the previous macOS version, which is currently being tracked here: #135

**Comment by gvnnz** [comment]
On a second thought, UPX might also be involved. Could you please try the fix reported [here](https://stackoverflow.com/a/64895860/3296421) and see if it works for you?

**Comment by nay-seven** [comment]
thanks, but don't work here, after upx install, i have now this message " you own 10.14.6 , this app needs 10.15 to work..

**Comment by gvnnz** [comment]
OK so it's a problem of missing backward compatibility. Let's see if we can target 10.14 in the next release.

**Comment by Xpktro** [comment]
This is still persisting. Just downloaded latest Giada and downgraded up to Talisman (v0.16.3) to get a working distribution :(

**Comment by gvnnz** [comment]
@Xpktro are you getting the "this app needs 10.15 to work..." message?

**Comment by Xpktro** [comment]
@gvnnz Yes, tried to compile it myself and got stuck with compilation errors displaying: `error: 'function' is unavailable: introduced in macOS 10.15`. I was able to run the v0.16.3 version without propblems, tho.

**Comment by gvnnz** [comment]
We really don't have the required tools to properly test this at the moment. I'm going to postpone it to the 1.future milestone.
